tests/test_download.py

```diff
@@ -44,8 +44,19 @@ def test_invalid_cloud_prefix(self, remote_local_file: Any):
         with pytest.raises(ValueError):
             mock_remote_filepath, mock_local_filepath = remote_local_file(
                 cloud_prefix='aaazure://')
-            download_from_azure(mock_remote_filepath, mock_local_filepath)
-            download_from_azure_datalake(mock_remote_filepath, mock_local_filepath)
+            downloader = AzureDownloader()
```

Comment:
instead of instantiating classes directly in these tests, can we create the clients with `CloudDownloader.get(remote)` every time?
Just want to make sure that's in the tests since that's how we call it in the code.

Reply:
We can also have a separate test which validates the prefix and `get(remote)` method

Reply:
@snarayan21 those tests are basically what we have before, but we do have a separate test which validates the prefix and the `get(remote)` method itself. See `test_getting_downloader()`

Reply:
I think another thing is that with this new API, we have 2 levels of validation.

1. Validation when we are getting a downloader
2. Validation when we are downloading a file.

I think these tests are useful for (2) as we can have scenarios someone sending in a mangled prefix/scheme for the remote path. One good example I can think of might probably be related to DBFS-UC since it requires the top most folder be `Volume`

Reply:
Fair. ok then this is fine w me